lib/client/group/mediator-topics/read.js

```diff
@@ -0,0 +1,33 @@
+var CONSTANTS = require('../../../constants');
+
+/**
+ * Initialsing a subscriber for Reading groups.
```

Comment:
Comment not matching method. Maybe we can just remove comments as they have very little value over the actual method name.